test/clt-tests/core/test-character-transformation-when-sending-to-buddy.rec

```diff
@@ -0,0 +1,50 @@
+––– input –––
+sed -i '/data_dir = \/var\/lib\/manticore/a buddy_path = manticore-executor -n /usr/share/manticore/modules/manticore-buddy/src/main.php --debug' /etc/manticoresearch/manticore.conf
+––– output –––
+––– input –––
+rm -f /var/log/manticore/searchd.log; searchd --stopwait > /dev/null; searchd; if timeout 10 grep -qm1 '\[BUDDY\] started' <(tail -n 1000 -f /var/log/manticore/searchd.log); then echo 'Buddy started!'; else echo 'Timeout or failed!'; cat /var/log/manticore/searchd.log;fi
```

Comment:
Is "cat /var/log/manticore/searchd.log" important?

Reply:
I left this in so that in case the test fails, we can see the errors in the logs right away.